packages/react-native/scripts/cocoapods/__tests__/new_architecture-test.rb

```diff
@@ -407,10 +407,10 @@ def test_newArchEnabled_whenRCTNewArchEnabledIsSetTo0_returnFalse
         assert_false(is_enabled)
     end
 
-    def test_newArchEnabled_whenRCTNewArchEnabledIsNotSet_returnTrue
+    def test_newArchEnabled_whenRCTNewArchEnabledIsNotSet_returnFalse
```

Comment:
Do we need `# [macos` markers?

Reply:
Yes, I missed that since this was a revert. However, I need to make some other change because RNTester is still running with new arch after this change..

Reply:
Nevermind, re-running RNTester, I get the old arch. So I'll add tags